contrib/kafka/kafkaeventsource-operator/sample/03-service.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: serving.knative.dev/v1alpha1
+kind: Service
+metadata:
+  name: message-dumper
+spec:
+  runLatest:
+    configuration:
+      revisionTemplate:
+        spec:
+          container:
+            image: matzew/message_dumper
```

Comment:
Better to use the official Message Dumper image located at `gcr.io/knative-releases/github.com/knative/eventing-sources/cmd/message_dumper`.